selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/workspaces/AddOrImportProjectFormTest.java

```diff
@@ -303,6 +293,15 @@ public void checkProjectSamples() {
         EXPECTED_SAMPLES_WITH_DESCRIPTIONS.get(CONSOLE_SAMPLE_NAME),
         EXPECTED_CONSOLE_REPOSITORY_URL);
 
+    addOrImportForm.clickOnAddOrImportProjectButton();
```

Comment:
it would be better to have separate steps divided by empty rows here from the readability point